dev/io.openliberty.jakarta.jstl.2.0/bnd.bnd

```diff
@@ -9,11 +9,11 @@
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
-
--sub: *.bnd
-
 bVersion=1.0
 
+Bundle-SymbolicName: io.openliberty.jakarta.jstl.2.0; singleton:=true
+Bundle-Description: Jakarta Standard Tag Library, version 2.0
+
```

Comment:
I can't comment on any lines below, but could you update the import on line 23? 
It says 2.0.0 to 4.0.0, but that's incorrect. It should only go up to 3.0.0. It was a typo from the very beginning. 
`Import-Package: jakarta.servlet.jsp.*;version="[2.0.0,4.0.0)", \` 

Thanks Paul for spotting this error!

Reply:
I created https://github.com/OpenLiberty/open-liberty/issues/21561 for us to follow up on.

Reply:
Yeah I did it the right way in the other location.  I didn't want to change what you had here, but it should be easy for you to do in your other issue.

Reply:
Looks to me. Thanks!